.github/workflows/pull_request.yaml

```diff
@@ -6,7 +6,7 @@ on:
       - main
 
 permissions:
-  contents: read
+  contents: write # to be able to write comments
```

Comment:
Need to check if this is safe

Reply:
removed for now. Opened for the future: https://github.com/benchmark-action/github-action-benchmark/issues/197